apps/web/src/hooks/useMintContractWithQuantity.tsx

```diff
@@ -59,7 +62,7 @@ export default function useMintContractWithQuantity({
     async (contract: any) => {
       if (contract && address) {
         try {
-          const balance = await contract.balanceOfType(tokenId, address);
+          const balance = await contract.read.balanceOfType([tokenId, address]);
```

Comment:
all ABI methods are on `read` or `write` properties on the contract now

and ABI method args are passed in as an array